subgraphs/aura-finance/src/common/constants.ts

```diff
@@ -6,6 +6,9 @@ import { Address, BigDecimal, BigInt } from "@graphprotocol/graph-ts";
 
 export const PROTOCOL_NAME = "Aura Finance";
 export const PROTOCOL_SLUG = "aura-finance";
+export const PROTOCOL_SCHEMA_VERSION = "1.3.0";
+export const PROTOCOL_SUBGRAPH_VERSION = "1.0.1";
```

Comment:
You can remove these right? It looks like you are using the generated version from the deployment.json, so you do not need to keep these. Also, is the version supposed to be 1.0.2?